### PR TITLE
OMCONNECT-911: Create New Order doc etc. changes

### PR DESCRIPTION
**Pull request by shrielenee**
## Purpose of this pull request

This pull request (PR) adds more references to the Create New Order option in the Sales > Orders section of the documentation. It also makes some other revisions for usability and cross-linking purposes.

Another PR for OMCONNECT-819 is forthcoming to build on these changes.

Reference preview build 13 to view these changes.

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/sales/orders-workspace.html
- https://docs.magento.com/user-guide/sales/orders.html
- https://docs.magento.com/user-guide/customers/customer-account-create-order.html
- https://docs.magento.com/user-guide/stores/configuration-overview.html

## Links to Magento source code or PRs

- ...
- ...

### CODE REVIEW

**Review by danidelcar — APPROVED** [review]

**Review by danidelcar — APPROVED** [review]

File: src/sales/orders.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 title: Orders
 ---
 
-The Orders workspace lists all current orders in descending order from latest purchase date. Each row in the grid represents an order, and each column represents an attribute, or data field. By default, orders display in descending order from latest to earliest purchase date. Use the standard [controls]({% link stores/admin-grid-controls.md %}) to sort and filter the list, find orders, and apply [actions]({% link stores/admin-actions-control.md %}) to selected orders. You can view existing orders, and create new orders. The tabs above the pagination controls can be used to filter the list, change the default view, change and rearrange columns, and export data.
+The Orders workspace lists all current orders in descending order from latest purchase date. Each row in the grid represents an order, and each column represents an attribute, or data field. By default, orders display in descending order from latest to earliest purchase date.
```
Comment (leslietilling) [inline src/sales/orders.md:5]:
```suggestion
The Orders workspace lists all current orders, where each row in the grid represents an order, and each column represents an attribute, or data field. By default, orders are displayed in descending order from latest to earliest purchase date.
```

File: src/sales/orders.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
+
+Use the standard [controls]({% link stores/admin-grid-controls.md %}) to sort and filter the list, find orders, and apply [actions]({% link stores/admin-actions-control.md %}) to selected orders. The tabs above the pagination controls can be used to filter the list, change the default view, change and rearrange columns, and export data.
+
+In the [Order workspace]({% link sales/orders-workspace.md %}) you can view existing orders and create new orders (you can also create new orders in the [Customer view]({% link customers/customer-account-create-order.md %})).
```
Comment (leslietilling) [inline src/sales/orders.md:9]:
```suggestion
In the [Order workspace]({% link sales/orders-workspace.md %}), you can view existing orders and create new orders. You can also create new orders from the [Customer view]({% link customers/customer-account-create-order.md %}).
```

**Review by leslietilling — APPROVED** [review]
Some minor suggested edits

**Comment by shrielenee** [comment]
running tests

**Comment by shrielenee** [comment]
running tests

**Pull request merged**